frontend/common/project.js

```diff
@@ -12,4 +12,5 @@ module.exports = global.Project = {
     chargebee: {
         site: 'flagsmith-test',
     },
+    zendesk: "427919a8-1cb2-40d6-9a9f-4214bf577ddd"
```

Comment:
We don't want to hard code this in case open source people start hitting our chat?

Reply:
was left from testing